presentation/src/main/java/com/stop/ui/mission/MissionFragment.kt

```diff
@@ -214,20 +222,13 @@ class MissionFragment : Fragment(), MissionHandler {
         private const val MINUS = ""
         private const val LEFT_TIME = 60
 
-        private const val FAKE_USER_FILE_PATH = "fake_user_path"
-
-        private const val PERSON_LINE = "person_line"
-        private const val PERSON_LINE_COLOR = Color.MAGENTA
         private var PERSON_LINE_NUM = 0
 
-        private const val BUS_LINE = "bus_line"
-        private const val BUS_LINE_COLOR = Color.BLUE
         private var BUS_LINE_NUM = 0
 
         private val INIT_LOCATION = Location(0.0, 0.0)
 
-        private const val BUS_MARKER = "marker_bus_pin"
-        private const val BUS_MARKER_IMG = R.drawable.ic_baseline_directions_bus_32
+        private const val SECOND_UNIT = 1000
```

Comment:
```suggestion
        private const val SECOND_UNIT = 1_000
```
코틀린에서는 천 단위로 구분자 `_`를 넣어줄 수 있습니다.

Reply:
오 Underscore 처음 알았네요! 👍 
https://github.com/boostcampwm-2022/android10-PlzStop/pull/58/commits/ae985104711c5ca6e55532122392c9272dfa231a